rhba-dev-env/vagrant/provisioning/roles/setup_env/tasks/main.yml

```diff
@@ -0,0 +1,205 @@
+---
+- fail:
+    msg: "This role is only supported for Fedora hosts"
+  when: ansible_distribution != 'Fedora'
+
+- name: upgrade all packages
+  become: true
+  become_user: root
+  dnf:
+    name: "*"
+    state: latest
+- name: install the latest version of docker
+  become: true
+  become_user: root
+  dnf:
+    name: docker
+    state: latest
+
+- name: "docker storage configurations"
+  become: true
+  become_user: root
+  copy:
+    dest: /etc/sysconfig/docker-storage-setup
+    content: |
+          DEVS=/dev/sdb
+          VG=docker-vg
+
+- name: "docker storage setup"
+  become: true
+  become_user: root
+  command: docker-storage-setup 
+
+- name: "enable docker service"
+  become: true
+  become_user: root
+  service:
+   name: docker
+   enabled: yes
+
+- name: "start docker service"
+  become: true
+  become_user: root
+  service:
+   name: docker
+   state: started
+
+- name: ensure group "docker" exists
+  become: true
+  become_user: root
+  group:
+    name: docker
+    state: present
+
+- name: adding user vagrant to group docker
+  become: true
+  become_user: root
+  user:
+    name: vagrant
+    groups: docker
+    append: yes
+
+- name: add new docker configuration
+  become: true
+  become_user: root
+  template:
+    src: "registries.j2"
+    dest: /etc/containers/registries.conf
+
+- name: install cekit
+  become: true
+  become_user: root
+  dnf:
+    name: 'http://download-ib01.fedoraproject.org/pub/fedora/linux/updates/30/Everything/x86_64/Packages/c/cekit-3.2.0-1.fc30.noarch.rpm'
```

Comment:
@mmagnanijr I'm assuming this full URL to this rpm is being used because we can't use cekit-3.3.x yet, right?

Reply:
@errantepiphany you're right, currently it is not possible to build images with cekit-3.3.x due to incompatibilities.